operational/azure/total_instance_usage_report/azure_total_instance_usage_report.pt

```diff
@@ -0,0 +1,498 @@
+name "Azure Usage Report - Instance Time Used"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy produces a usage report showing the amount of time used for each Azure Virtual Machine/Instance Family. See the [README](https://github.com/flexera-public/policy_templates/tree/master/operational/azure/total_instance_usage_report) and [docs.flexera.com/flexera/EN/Automation](https://docs.flexera.com/flexera/EN/Automation/AutomationGS.htm) to learn more."
+long_description ""
+severity "low"
+category "Operational"
+default_frequency "monthly"
+info(
+  version: "1.0.0",
+  provider: "Azure",
+  service: "Compute",
+  policy_set: "Usage Report"
+)
+
+###############################################################################
+# Parameters
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  category "Policy Settings"
+  label "Email Addresses"
+  description "Email addresses of the recipients you wish to notify."
+  default []
+end
+
+parameter "param_months_back" do
+  type "number"
+  category "Policy Settings"
+  label "Months Back"
+  description "How many months to go back in the chart/incident."
+  min_value 1
+  max_value 12
+  default 12
+end
+
+parameter "param_unit_instance" do
+  type "string"
+  category "Policy Settings"
+  label "Instance Unit"
+  description "Instance unit to normalize usage against. See README for more details."
+  allowed_values "Normalized Units (NFUs)", "vCPUs", "Memory (GiB)"
```

Comment:
"Normalized Units (NFUs)" - This parameter value (and the README) should be renamed from NFUs (which is AWS specific) to something more reflective of the Azure platform.